.gitlab-ci.yml

```diff
@@ -0,0 +1,73 @@
+# Setup a cache to cache job parts between jobs to ensure faster builds
```

Comment:
why we need this file if we host the repo in github?

Reply:
I have removed that file.